src/style.scss

```diff
@@ -0,0 +1,186 @@
+/* #region root */
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+$white: #fff;
+
+:root {
+  --main-accent: #{$main-accent};
+  --secondary: #{$secondary};
+  --blue-accent: #{$blue-accent};
+  --white: #{$white};
+}
+
+body {
+  margin: 0;
+  display: flex;
+}
```

Comment:
when your file structure will be as one in checklist, you can't reset those styles in index.scss